tests/appsec/iast/sink/test_hsts_missing_header.py

```diff
@@ -0,0 +1,65 @@
+# Unless explicitly stated otherwise all files in this repository are licensed under the the Apache License Version 2.0.
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2021 Datadog, Inc.
+
+from utils import context, coverage, released, missing_feature, weblog
+from .._test_iast_fixtures import SinkFixture
+
+
+@coverage.basic
+@released(dotnet="?", golang="?", php_appsec="?", python="?", ruby="?", nodejs="?")
+@released(
```

Comment:
You must remove this line, as it is declared her : https://github.com/DataDog/system-tests/pull/1561/files#diff-3c1780967977e9493a0eaa12620a14e42648d843b310d3481b758a8250ad600bR6

Reply:
applied

Reply:
fixed